src/components/MediaSettings/VideoBackgroundEditor.vue

```diff
@@ -23,10 +23,10 @@
 	<div class="background-editor">
 		<button key="clear"
 			class="background-editor__element"
-			:class="{'background-editor__element--selected': selectedBackground === 'clear'}"
-			@click="handleSelectBackground('clear')">
+			:class="{'background-editor__element--selected': selectedBackground === 'none'}"
+			@click="handleSelectBackground('none')">
 			<Cancel :size="20" />
-			{{ t('spreed', 'Clear') }}
+			{{ t('spreed', 'None') }}
```

Comment:
This is not really translatable for translators without further context.

Please see the Vue section of https://docs.nextcloud.com/server/latest/developer_manual/basics/front-end/l10n.html#provide-context-hints-for-translators